task/04-arrays-tasks.js

```diff
@@ -304,7 +340,15 @@ function get3TopItems(arr) {
  *   [ 1, '2' ] => 1
  */
 function getPositivesCount(arr) {
-   throw new Error('Not implemented');
+   let result=0
+   for (let i = 0; i<arr.length;i++){
+      if (typeof (arr[i])==="string"){
+         i++
+      }else if (arr[i]>0){
+         result+=1
+      }
+   }
+   return result
```

Comment:
Please, try to find solution with no loops
Use filter method for positives, and return its length